seatunnel-connectors/seatunnel-connector-spark-jdbc/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc2/DefaultSource.scala

```diff
@@ -102,6 +102,6 @@ class DefaultSource extends CreatableRelationProvider with RelationProvider
       conn.close()
     }
 
-    createRelation(sqlContext, parameters)
+    JDBCRelation(Array.empty, new JDBCOptions(parameters))(sqlContext.sparkSession)
   }
```

Comment:
@zhaomin1423 It doesn't seem like this needs to be changed.

Reply:
There are a few redundant operations, for example, get schema and get partitions.

Reply:
@zhaomin1423 I think it's better to keep it as it is, because this piece is not the cause of the error, I'd like to hear what other people think

Reply:
> @zhaomin1423 I think it's better to keep it as it is, because this piece is not the cause of the error, I'd like to hear what other people think

Change the code, but leave a comment which tell other why do like that. Maybe this way is better?

Reply:
Thanks, I leave a comment.

Reply:
@zhaomin1423 I still do not recommend modifying this code, please refer to: `https://github.com/apache/spark/blob/master/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JdbcRelationProvider.scala`

Reply:
@zhaomin1423 This is the official spark code, I think you need to revert your changes

Reply:
I merged first, I'll restore it later